.github/workflows/pipedream-sdk-test.yaml

```
@@ -18,15 +18,30 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      - uses: pnpm/action-setup@v4.0.0
+        with:
+          version: 9.14.2
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        run: |
```

actionlint (1.7.3)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)